gh-actions/rust/code-sanity/action.yaml

```diff
@@ -0,0 +1,68 @@
+name: Rust code sanity check
+description: Checks code against our desktop Rust code quality and process standards.
+
+inputs:
+  token:
+    required: true
+    description: Token required to authenticate certain Rust steps
```

Comment:
I think we should be more explicit what this token is, is it the github token? (spoiler alert: I know it’s that one, but without looking at our previous integration version in authd, I would not :p)